platforms/maven/pom.xml

```diff
@@ -0,0 +1,247 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>opencv.org</groupId>
+    <artifactId>opencv</artifactId>
+    <packaging>bundle</packaging>
+    <version>3.1.0</version>
```

Comment:
Hi alalek. I had made a brief attempt to link the version to that which I extract using one of the bash scripts however Maven may insist on a hard-coded value rather than a property that can be set at build time. I will revisit this again though as I agree it should, ideally, not be hard coded.